authentik/stages/captcha/stage.py

```diff
@@ -27,6 +27,60 @@ class CaptchaChallenge(WithUserInfoChallenge):
     component = CharField(default="ak-stage-captcha")
 
 
+def get_friendly_captcha_error(error: str) -> str:
```

Comment:
Hmm I'm not sure if these error codes are the same for all the Captcha implementations we support. Also things like invalid/missing secret are probably not something we should disclose to end-users
I think we can probably fall back to `Invalid captcha response` as a generic error message

Reply:
Yeah, fair enough. I was caught up on how to help admins, but this is not the way, I'll add some logging for that.

I think it's reasonable to keep the ones where we advise end users to try again, and I'll make them even more friendly. I'll include all "recoverable-by-retry" error codes from [reCAPTCHA](https://developers.google.com/recaptcha/docs/verify#error_code_reference), [hCaptcha](https://docs.hcaptcha.com/#siteverify-error-codes-table) and [Turnstile](https://developers.cloudflare.com/turnstile/get-started/server-side-validation/#error-codes).